server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ACVRDownloadByCounty.java

```diff
@@ -50,7 +55,7 @@ public EndpointType endpointType() {
    */
   @Override
   public String endpointName() {
-    return "/acvr/:counties";
+    return "/acvr/county/:counties";
```

Comment:
Ah, `PUT` or `DELETE` is a better in this case, but in others it often makes sense to `POST` data to a particular entity to create a subresource. I probably just shouldn't have chosen the exact worst HTTP method to illustrate my point (-: